src/config/constants.ts

```diff
@@ -49,6 +49,8 @@ export const LAST_NAME_LABEL = 'Last Name';
 // Status management
 export const NONE_STATUS_TEXT =
   'You’re all set! Ready to start your application?';
+export const DEADLINE_STATUS_TEXT =
```

Comment:
I don't think this is a status, can we move this under Application management constants?